tests/js/spec/views/settings/projectPerformance.spec.jsx

```diff
@@ -0,0 +1,91 @@
+import {mountWithTheme} from 'sentry-test/enzyme';
+
+import ProjectPerformance from 'app/views/settings/projectPerformance/projectPerformance';
+
+describe('projectPerformance', function () {
+  const org = TestStubs.Organization({features: ['project-transaction-threshold']});
+  const project = TestStubs.ProjectDetails();
+  let getMock, postMock, deleteMock;
+
+  beforeEach(function () {
+    MockApiClient.clearMockResponses();
+    getMock = MockApiClient.addMockResponse({
+      url: '/projects/org-slug/project-slug/transaction-threshold/configure/',
+      method: 'GET',
+      body: {
+        id: project.id,
+        threshold: '300',
+        metric: 'duration',
+      },
+      statusCode: 200,
+    });
+    postMock = MockApiClient.addMockResponse({
+      url: '/projects/org-slug/project-slug/transaction-threshold/configure/',
+      method: 'POST',
+      body: {
+        id: project.id,
+        threshold: '400',
+        metric: 'lcp',
+      },
+      statusCode: 200,
+    });
+    deleteMock = MockApiClient.addMockResponse({
+      url: '/projects/org-slug/project-slug/transaction-threshold/configure/',
+      method: 'DELETE',
+      statusCode: 200,
+    });
+  });
+
+  it('renders the fields', async function () {
+    const wrapper = mountWithTheme(
+      <ProjectPerformance
+        params={{orgId: org.slug, projectId: project.slug}}
+        organization={org}
+        project={project}
+      />,
+      TestStubs.routerContext()
+    );
+    await tick();
+    expect(wrapper.find('input[name="threshold"]').prop('value')).toBe('300');
+    expect(wrapper.find('input[name="metric"]').prop('value')).toBe('duration');
+    expect(getMock).toHaveBeenCalled();
```

Comment:
I prefer `toHaveBeenCalledTimes(1)` so we don't slide on the number of requests either (and it will catch if you mocks aren't properly clearing, too)

Reply:
addressed in daf94f8052238644042533691a9de8f10d8ae7a1